databricks/koalas/tests/test_dataframe_conversion.py

```diff
@@ -70,3 +70,34 @@ def test_to_html(self):
             """)
         got = self.strip_all_whitespace(self.kdf.to_html(max_rows=2))
         self.assert_eq(got, expected)
+
+    def test_to_excel(self):
+        pdf = self.pdf
+        kdf = self.kdf
+        excel_writer = "output.xlsx"
+
+        self.assert_eq(kdf.to_excel(excel_writer), pdf.to_excel(excel_writer))
```

Comment:
should we assert the output.xlsx is the same, rather than asserting on the return value?

Reply:
Doing that.

Reply:
@rxin, should I load the values of the generated excel files in 2 dataframes and check the values?

Reply:
Yes that'd work.

Reply:
Make sure you write it to a temporary location that's unique too.

Reply:
@rxin, I have made the changes you asked for.
Please have a look.